docs/development/releasing.rst

```diff
@@ -396,6 +420,8 @@ The procedure for this is straightforward:
 
 #. Repeat the above steps for the astropy-helpers, using the same version series.
 
+#. Run the tests twice in a row without cleaning anything up in between.
```

Comment:
I removed 32-bit testing from here, but you still want to keep this, right?

Reply:
sure 🤷‍♀ 

(confession: most of the time I don't run them at this point any more as they take forever and in "CI we trust" 🙈 )

Reply:
@astrofrog , you requested this. What say you?

Reply:
@astrofrog and I agreed that this can be removed from release instruction and added to CI directly at a later time.